serde_v8/magic/buffer.rs

```diff
@@ -14,19 +14,26 @@ use crate::magic::transl8::impl_magic;
 pub enum MagicBuffer {
   FromV8(ZeroCopyBuf),
   ToV8(Mutex<Option<Box<[u8]>>>),
+  Temp(Vec<u8>),
```

Comment:
Would be nice to get a comment explaining what this "Temp" means and when it's used